rtf_vis_tool/src/Components/PCViewer.js

```diff
@@ -75,34 +73,12 @@ function PCViewer(props) {
                 <PointMesh  
                     position={[pointArr[1], pointArr[2], pointArr[3]]}  
                     color={`rgb(${pointArr[4]}, ${pointArr[5]}, ${pointArr[6]})`} 
-                    size={pointSizeArr}/>
+                    size={[pointRadius]}/>
```

Comment:
cool -- so we don't need the `updatePointSizes` function anymore?

Reply:
nope. the `useEffect` function will now run every time `pointRadius` is changed in the slider, meaning `loadCOLMAPPointCloud` will update the point sizes. So `updatePointSizes` isn't neccessary anymore.